PowerShell_Scripts/FolderSynchronisation_edited+func_2ed.ps1

```diff
@@ -0,0 +1,27 @@
+﻿Clear-Host
+
+Write-Host "We will ask you to indicate the path for folder 1 and folder 2"
+function Path($TestFolderPath) {
```

Comment:
While correct from the syntactic perspective, this line has a few issues:
1. `$TestFolderPath` parameter isn't used and therefore not needed. On a related note, it can be a good practice to stick to [declaring parameters in a Param block within function's braces](https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_functions?view=powershell-7.1#functions-with-parameters), this comes handy in bigger scripts where we want to specify parameter types or properties (e.g. make them mandatory).
2. `Path` is not an indeal name for a function - mainly because it implies object isntead of action/operation. Something like `GetPath` will be a better approach. Better yet, use [Approved Verbs for PowerShell commands](https://docs.microsoft.com/en-us/powershell/scripting/developer/cmdlet/approved-verbs-for-windows-powershell-commands?view=powershell-7.1), they can be really helpful in the long run (although sometime inflexible).

Reply:
Thanks, got it!

```suggestion
function GetPath {
    param($TestFolderPath)

    Write-Host "Please, specify the path"
    $TestFolderPath = Read-Host

    return $TestFolderPath
}
```